tests/plugins/table/table.js

```diff
@@ -145,6 +145,39 @@
 				'</table>' );
 
 			assert.isTrue( /border="1"/.test( bot.editor.getData() ), 'Border attribute should be one' );
+		},
+		'test changing headers to none from dialog': function() {
+			var bot = this.editorBots.editor;
+			var editor = this.editors.editor;
+
+			bot.setHtmlWithSelection(
+				'<table border="1" cellspacing="1" cellpadding="1" style="width:500px;">' +
+					'<thead>' +
+						'<tr>' +
+							'<th scope="row">' +
+							'<br>^' +
+							'</th>' +
+							'<th scope="col">' +
+							'<br>' +
+							'</th>' +
+						'</tr>' +
+					'</thead>' +
+					'<tbody>' +
+					'</tbody>' +
+				'</table>'
+			);
+
+			bot.contextmenu( function() {
+				//Dialog is nested in context menu in order to make changes to properties of existing table instead of creating new one.
```

Comment:
You don't really need to do that using context menu. Instead you can call table properties command directly.

Missing space before "dialog".